<?xml version="1.0"?>
<launch>
    <include file="$(find aruco_proxy_pose_estimation)/launch/apriltag_grid_detector.launch">
        <arg name="config_file" value="$(find corsmal_setup_bringup)/config/apriltag_grid_detector_config_right_azure.yaml"/>
    </include>
    <node pkg="corsmal_setup_bringup" type="calibrate_right_camera.py" name="calibrate_right_camera" />
</launch>